package.json

```diff
@@ -51,15 +51,15 @@
     "i18n:extract": "lingui extract",
     "i18n:compile": "lingui compile",
     "postinstall": "husky install",
-    "betterer": "betterer"
+    "betterer": "betterer --cache"
   },
   "grafana": {
     "whatsNewUrl": "https://grafana.com/docs/grafana/next/whatsnew/whats-new-in-v8-4/",
     "releaseNotesUrl": "https://grafana.com/docs/grafana/next/release-notes/"
   },
   "lint-staged": {
     "*.{js,ts,tsx}": [
-      "betterer precommit",
+      "betterer --cache precommit",
```

Comment:
yeah, it'll run against the entire codebase whenever you commit a js/ts/tsx file. we need to do that initially to get the full count of the errors. on subsequent commits it uses the cache, so it'll only ever need to run against js/ts/tsx files that have changed and everything else it will grab from the cache.